src/frontend/src/lib/components/wizards/migration/views/EnterIdentityNumber.svelte

```diff
@@ -13,6 +15,11 @@
   let { onSubmit, isAuthenticating }: Props = $props();
 
   let identityNumber = $state<string | undefined>(undefined);
```

Comment:
Why do me make this undefined? Since it's an input box I'd instead suggest to simplify to:

```suggestion
  let identityNumber = $state("");
```

Reply:
good point